src/item.cpp

```diff
@@ -825,7 +825,41 @@ std::string Item::getDescription(const ItemType& it, int32_t lookDistance,
 				tmpSubType = item->getSubType();
 			}
 
-			s << ". " << (it.stackable && tmpSubType > 1 ? "They" : "It") << " can only be used with";
+			s << ". " << (it.stackable && tmpSubType > 1 ? "They" : "It") << " can only be used by ";
+
+			VocSpellMap vocMap = g_spells->getRuneSpell(it.id)->getVocMap();
```

Comment:
1) getRuneSpell() can return null
2) is there any reason you're copying the map instead of holding it by const reference?